paddle/fluid/memory/allocation/allocator.h

```diff
@@ -132,9 +152,12 @@ class Allocator {
   // True if the `Allocate` is thread safe.
   virtual bool IsAllocThreadSafe() const;
 
+  // This function should not be called outside
```

Comment:
Because some decorated allocator would call something like `underlying_allocator_->Free(...)`. Comments above is saying that we should not call this method outside the implementation of allocator.